scripts/webui_streamlit.py

```diff
@@ -1118,21 +1124,27 @@ def process_images(
 				while(torch.cuda.memory_allocated()/1e6 >= mem):
 					time.sleep(1)
 
-			if variant_amount == 0.0:
+			if noise_mode == 1:
+				# TODO params for find_noise_to_image
+				x = torch.cat(batch_size * [find_noise_for_image.find_noise_for_image(
+					st.session_state["model"], st.session_state["device"],
+					init_img.convert('RGB'), '', find_noise_steps, 0.0, normalize=True,
+					generation_callback=generation_callback,
```

Comment:
should remove the generation callback param. it's not setup to work like this, I just liked having the preview.
you'd see "30/30" and it just keeps running when the find_noise_steps is >100

perhaps the generation callback can be reworked a tad to be more flexible

Reply:
Oh, you can do like I did on the txt2vid tab, [check this](https://github.com/sd-webui/stable-diffusion-webui/blob/ba63fd7004d397627b9bfd0ab0b4eafc65fe677d/scripts/webui_streamlit.py#L660).